File: 782-jewels-and-stones/jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int sum = 0;
        int arr[58];
        for (int i = 0; i < jewels.length(); i++) {
            arr[jewels[i] - 'A'] = 1;
        }
        for (int i = 0; i < stones.length(); i++) {
            sum += arr[stones[i] - 'A'];
        }
        return sum;
    }
};

File: 782-jewels-and-stones/jewels-and-stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int count = 0;
        List<Character> al = new ArrayList<>();
        for (int i = 0; i < jewels.length(); i++) {
            if (!al.contains(jewels.charAt(i))) {
                al.add(jewels.charAt(i));
            }
        }
        for (int i = 0; i < stones.length(); i++) {
            if (al.contains(stones.charAt(i))) {
                count++;
            }
        }
        return count;
    }
}